### PR TITLE
Envoy-update entry method type

### PR DESCRIPTION
**Pull request by iannater**
Correcting a type error

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/envoy/apiDefinition.swagger.json [hunk 1]
```diff
@@ -2708,7 +2708,7 @@
                               "type": "object",
                               "properties": {
                                 "type": {
-                                  "type": "string",
+                                  "type": "object",
```
Comment (Amjed-Ayoub) [inline certified-connectors/envoy/apiDefinition.swagger.json:2711]: @iannater The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. can you please confirm that no existing flows and users will be affected by this change?

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/envoy/apiDefinition.swagger.json (hunk 1 above)
Comment (iannater) [reply certified-connectors/envoy/apiDefinition.swagger.json:2711]: @Amjed-Ayoub Yes! This field is currently breaking existing flows and we are making this change to fix it.

**Comment by iannater** [comment]
@Amjed-Ayoub I wanted to check and make sure there wasn't anything else you needed from me on this?

**Comment by bheddens** [comment]
@Amjed-Ayoub what are the next steps to getting this change introduced? We are aware that it's a breaking change, but it's also currently broken for customers so the update needs to happen. Thanks in advance!

**Comment by Amjed-Ayoub** [comment]
@iannater thank you so much for the clarification. and for confirming this change.

**Review by Amjed-Ayoub — APPROVED** [review]
Hello @iannater,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Comment by iannater** [comment]
@Amjed-Ayoub if there is another change that we want to make not related to this bug but another one can I update this PR or should I do this as a separate PR?

**Comment by iannater** [comment]
@Amjed-Ayoub Do I need to wait till a previous change is fully deployed before I can update the connector? I can't seem to edit this right now but noticed that it isn't "certified" yet and still in "Production Deployment".


image

image

**Comment by Amjed-Ayoub** [comment]
Hello @iannater yes you are correct. but we certified and completed the current deployment. you may make a new submission :-).

Thank you so much.

**Comment by iannater** [comment]
@Amjed-Ayoub I am getting this error? 
image

**Comment by Amjed-Ayoub** [comment]
@iannater can you please share the commit ID you are using? is it "45fac2b908bb6b697559cfbc7527a83c7ea8f582"?

**Comment by iannater** [comment]
@Amjed-Ayoub  Yes, that is correct. I grabbed it from here. 

image

**Comment by Amjed-Ayoub** [comment]
@iannater please try with this "b418fa39b94d818d7fb2cf794147f69c14d46ccc"

**Comment by iannater** [comment]
Hey @Amjed-Ayoub still getting an error. 
image

**Comment by iannater** [comment]
Hey @Amjed-Ayoub still getting an error
image

**Comment by Amjed-Ayoub** [comment]
@iannater one last time please, may I ask you please to download the connector files from this repository and add your icon and the intro.md file and have them in a .zip format. Then please try to submit using the second option "Submit by uploading a .zip archive".
I hope that helps.
Thank you so much again for your patience.

**Comment by iannater** [comment]
@Amjed-Ayoub I downloaded the connector file that I modified and uploaded it with the other documents and got this error?
image

**Comment by Amjed-Ayoub** [comment]
@iannater can you please start an email thread with us regarding this. and please share the .zip folder in the email.
I look forward to hearing from you.
Thank you so much again.

**Comment by iannater** [comment]
@Amjed-Ayoub what email should I be using for this thread?

**Comment by Amjed-Ayoub** [comment]
> @Amjed-Ayoub what email should I be using for this thread?

Connector Partner Mgmt Team <ConnectorPartnerMgmtTeam@service.microsoft.com>

**Comment by iannater** [comment]
@Amjed-Ayoub I have sent 4 emails now and no one has responded to me. What am I supposed to do?

**Comment by Amjed-Ayoub** [comment]
Hello @iannater thank you so much but I have responded to your email ever time and provided a zip folder to make the submission with. please check you spam folder. 
image

**Comment by SaiChander-MS** [comment]
@iannater 
Sorry for delay in response. We are updating the status on ISV studio activity control. Currently it is waiting on partner for verifying changes on Preview. Could you please check ISV activity control and respond accordingly.

Thanks.

**Comment by SaiChander-MS** [comment]
@iannater Gentle reminder to test your connector in Preview and update us in ISV activity control, so that we can start with deployment process.

**Comment by iannater** [comment]
Hello,
Thanks for the reminder, it's on my list and I am hoping to get to it this week.
Ian  





On Tue, Aug 1, 2023 at 1:35 AM SaiChander-MS ***@***.***>  wrote:


@iannater  Gentle reminder to test your connector in Preview and update us in
ISV activity control, so that we can start with deployment process.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: 
***@***.***>



Ian Nater
Customer Support Engineer
***@***.***

**Comment by iannater** [comment]
Hello,
We tested this and it is working. Please push it to prod.
Thanks,

Ian  





On Tue, Aug 1, 2023 at 9:22 AM Ian Nater ***@***.***>  wrote:
Hello,
Thanks for the reminder, it's on my list and I am hoping to get to it this week.
Ian  





On Tue, Aug 1, 2023 at 1:35 AM SaiChander-MS ***@***.***>  wrote:


@iannater  Gentle reminder to test your connector in Preview and update us in
ISV activity control, so that we can start with deployment process.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: 
***@***.***>



Ian Nater
Customer Support Engineer
***@***.***




Ian Nater
Customer Support Engineer
***@***.***

**Pull request merged**